.env.examples

```
@@ -0,0 +1,15 @@
+NODE_ENV=
+
+NEXT_PUBLIC_BASE_URL=
+
+MONGODB_URI=
+
+BETTER_AUTH_SECRET=
+
+BETTER_AUTH_URL= 
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)